python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -444,6 +444,9 @@ def __get__(self, obj, cls):
 
         # Since we couldn't find the information in the cache, we will need to fully load the object
 
+        if name in obj.__dict__:
+            return obj.__dict__[name]
```

Comment:
There should be no situation where a `Descriptor` getter is getting things from the object's `__dict__`. They are completely separate things. A given `Descriptor` should only look at its entry in `_data` and no where else (modulo lazy-loading).